src/RageUtil.cpp

```diff
@@ -124,6 +124,16 @@ float fmodfp(float x, float y)
 
 int power_of_two( int input )
 {
+	if (input == 0)
+	{
+		return 0;
+		LOG->Trace("power_of_two: input is %d, returning 0", input);
+	}

```

Comment:
0 is not a power of two. This contradicts the function's name (and most likely the intended behavior judging by the uses). 
This function should return at least `1` (`2^0 = 1`). 

However, you should probably check if the places that use the output value don't also expect this to be a power of two that's also a multiply of two at the same time. That would limit the expected return values to `2+` (`2^1`+).